config/config.default.yaml

```
@@ -452,6 +452,7 @@ sector:
   hydrogen_fuel_cell: true
   hydrogen_turbine: false
   SMR: true
+  SMR_cc: false
```

```suggestion
  SMR_cc: true
```
To keep current defaults.